File: app/src/main/java/com/example/james/rms/Core/ServePath/ReceivingItemServePath.java
package com.example.james.rms.Core.ServePath;

import com.example.james.rms.NetWork.ServeProfile;

/**
 * Created by jamie on 2017/6/10.
 */

public class ReceivingItemServePath {
    public static String findReceivingItemByPartyId(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getRe_item_findByPartyId();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String save(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getRe_item_save();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String saves(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getRe_item_save();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String deleteByProductId(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getRe_item_deleteByProductId();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String deleteByOrderId(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getRe_item_deleteByOrderId();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String delete(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getRe_item_delete();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String deletes(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getRe_item_deletes();
        String serve_path = serve+path;
        return serve_path;
    }

}

File: app/src/main/java/com/example/james/rms/Core/ServePath/ReceivingItemServePath.kt
package com.example.james.rms.Core.ServePath

import com.example.james.rms.NetWork.ServeProfile

/**
 * Created by jamie on 2017/6/10.
 */

object ReceivingItemServePath {
    val serve = ServeProfile.serve

    fun findReceivingItemByPartyId(): String {
        val path = ServeProfile.re_item_findByPartyId
        return serve + path
    }

    fun save(): String {
        val path = ServeProfile.re_item_save
        return serve + path
    }

    fun saves(): String {
        val path = ServeProfile.re_item_save
        return serve + path
    }

    fun deleteByProductId(): String {
        val path = ServeProfile.re_item_deleteByProductId
        return serve + path
    }

    fun deleteByOrderId(): String {
        val path = ServeProfile.re_item_deleteByOrderId
        return serve + path
    }

    fun delete(): String {
        val path = ServeProfile.re_item_delete
        return serve + path
    }

    fun deletes(): String {
        val path = ServeProfile.re_item_deletes
        return serve + path
    }

    fun deleteByReceivingIds(): String {
        val path = ServeProfile.re_item_deleteByReceivingIds
        return serve + path
    }
}
